src/Components/Users/UserProfile.tsx

```diff
@@ -181,6 +183,45 @@ export default function UserProfile() {
     },
   );
 
+  const setNewPasswordErrorCallFun = (value: string) => {
+    if (value == "") {
+      setNewPasswordError("");
+      return;
+    }
+    setNewPasswordError(
+      [
+        value.length >= 8
+          ? ""
+          : "Password should be at least 8 characters long",
+        value !== value.toUpperCase()
+          ? ""
+          : "Password should contain at least 1 lowercase letter",
+        value !== value.toLowerCase()
+          ? ""
+          : "Password should contain at least 1 uppercase letter",
+        /\d/.test(value) ? "" : "Password should contain at least 1 number",
+      ]
+        .filter(Boolean)
+        .join("\n"), // Join only the non-empty error messages
+    );
+  };
+
+  const changePasswordErrorsCallFun = (value: string) => {
+    if (value === "" || value === changePasswordForm.new_password_1) {
+      setChangePasswordErrors((prev) => ({
+        ...prev,
+        password_confirmation: "",
+      }));
+      return;
+    } else {
+      setChangePasswordErrors((prev) => ({
+        ...prev,
+        password_confirmation:
+          "Confirm password should match the new password!",
+      }));
+    }
+  };
+
```

Comment:
both the functions seems to be complicated to read than necessary. try to simplify the code and make it more readable.

Reply:
done